NAS-103439 docs for displaying plugin Admin Portals. (#1660)
freenas/freenas-docs: userguide/plugins.rst

means the plugin will not start when the system boots and must be
started manually.

In addition to the :guilabel:`Jail` name, the :guilabel:`Columns`
menu controls which information is displayed for each installed

> Plugins entry. Options include status, IPv4 address, IPv6 address,

Reviewer: Don't list the options that can be enabled, just mention that additional columns can be enabled or disabled with the Columns menu.  We should make this a text replacement, like |columns-desc|.

Click |ui-chevron-right| and :guilabel:`MANAGE` to open a management
or configuration screen for the application. For example, clicking
:guilabel:`MANAGE` for an installed Plex plugin opens the Plex

> web interface in a new browser tab. Plugins that have a management

Reviewer: ```suggestion
web interface in a new browser tab. Plugins with a management
```

> interface also display the IP address and port to that management page

Reviewer: ```suggestion
interface show the IP address and port to that page
```
